### PR TITLE
adding support for a new BDI SDK and new auth flavor

### PR DESCRIPTION
**Pull request by gullerya**

### CODE REVIEW

File: src/main/java/com/hpe/application/automation/tools/octane/AbstractResultQueueImpl.java [hunk 1]
```diff
@@ -24,116 +24,116 @@
 import java.io.*;
 
 /**
- * Created by benmeior on 11/21/2016.
+ * Created by benmeior on 11/21/2016
  */
 
 public abstract class AbstractResultQueueImpl implements ResultQueue {
```
Comment (m-seldin) [inline src/main/java/com/hpe/application/automation/tools/octane/AbstractResultQueueImpl.java]: Please add documentation to modified class to align with hp-aa coding style

File: src/main/java/com/hpe/application/automation/tools/octane/buildLogs/BdiConfigurationFetcher.java [hunk 2]
```diff
@@ -27,83 +28,86 @@
...
 
 import java.io.IOException;
 
 /**
- * Created by benmeior on 12/21/2016.
+ * Created by benmeior on 12/21/2016
  */
+
 @Extension
 public class BdiConfigurationFetcher extends AbstractSafeLoggingAsyncPeriodWork {
```
Comment (m-seldin) [inline src/main/java/com/hpe/application/automation/tools/octane/buildLogs/BdiConfigurationFetcher.java]: add documentation to class

File: src/main/java/com/hpe/application/automation/tools/octane/buildLogs/LogAbstractResultQueue.java [hunk 3]
```diff
@@ -23,12 +23,13 @@
 import java.io.IOException;
 
 /**
- * Created by benmeior on 11/21/2016.
+ * Created by benmeior on 11/21/2016
  */
+
 public class LogAbstractResultQueue extends AbstractResultQueueImpl {
```
Comment (m-seldin) [inline src/main/java/com/hpe/application/automation/tools/octane/buildLogs/LogAbstractResultQueue.java]: add documentation to class

**Review by m-seldin — COMMENTED** [review]
please supply a person for detailed code review and modify coding style comments

**Comment by gullerya** [comment]
documentations added
main review of the code logic was done by Adi Godkin

**Review by m-seldin — APPROVED** [review]

**Pull request merged**